core/src/main/resources/lib/layout/main-panel.jelly

```diff
@@ -30,7 +30,11 @@ THE SOFTWARE.
 
   <j:if test="${mode=='main-panel'}">
     <j:if test="${app.isQuietingDown()}">
-      <div id='shutdown-msg'>${%Jenkins is going to shut down}</div>
+      <div id='shutdown-msg'>${%Jenkins is going to shut down}
+        <j:if test="${app.getQuietingReason() != null}">
+          <br/> ${%Shut down reason}: ${app.getQuietingReason()}
```

Comment:
Should it rather go through the Markup formatter so that one can set hyperlinks & Co?

Reply:
Can you give me a hint how to do it?

Reply:
`Jenkins.get().getMarkupFormatter().translate(String)`, there might be a convenient utility in `Functions` for this but I'm not sure.

Would consider this out of scope for this one.